css/style.css

```diff
@@ -286,6 +299,10 @@ button.button-inline:hover {
 	overflow: hidden;
 	text-overflow: ellipsis;
 }
+.stack ul.card-list {
```

Comment:
no need for both `ul` and `.card-list` - you can target this element specifically with `.stack > ul` or simply `.card-list`. Best not to have too many  queries in CSS